### PR TITLE
Fixed fee calculation

### LINKED ISSUE #51: Fix fee calculation

### PR DESCRIPTION
**Pull request by DanielaIvanova**
resolves #51

### CODE REVIEW

File: aeternity.yaml [hunk 1]
```diff
@@ -16,6 +16,7 @@ mining:
     autostart: true
     expected_mine_rate: 1000
     beneficiary_reward_delay: 2
+    min_miner_gas_price: 1000000
```
Comment (thepiwo) [inline aeternity.yaml]: is this the protocol minimum?

File: apps/utils/lib/transaction.ex [hunk 2]
```diff
@@ -100,48 +115,32 @@ defmodule Utils.Transaction do
...
-    end
+  @spec try_post(
+          struct(),
+          String.t(),
+          String.t(),
+          non_neg_integer(),
+          tx_types(),
+          non_neg_integer()
+        ) :: {:ok, map()} | {:error, String.t()} | {:error, Env.t()}
+  def try_post(
```
Comment (thepiwo) [inline apps/utils/lib/transaction.ex]: why is the function renamed?

File: aeternity.yaml (hunk 1 above)
Comment (DanielaIvanova) [reply aeternity.yaml]: Yes, is by protocol.

File: apps/utils/lib/transaction.ex (hunk 2 above)
Comment (DanielaIvanova) [reply apps/utils/lib/transaction.ex]: This is a new function.

**Comment by Artur64** [comment]
We think so.

**Pull request merged**